### DISCUSSION TITLE
Root Break and Spell Resisted added to Overlay

### DISCUSSION THREAD

**Discussion by Oziki** [smasherprog/EqTool#117] [Ideas]
I'd really love to get "Root Break" and if a spell is resisted on the overlay tab similar to Charm Break.  I'd like to use this by itself instead of using it with Gina.  Thanks for the awesome program!

**Comment by smasherprog** [comment]
Sure ill add this to the to do list.. EZ to add

**Comment by morgs71** [comment]
Snare too possibly

**Comment by smasherprog** [comment]
Added root break spell resists
